automated_packaging/update_os_package.pl

```diff
@@ -9,10 +9,24 @@
 # Name of the repo is represented differently on logs and repos
 my $github_repo_name = "citus";
 my $log_repo_name = "Citus";
+my $version_suffix = ".citus";
```

Comment:
We ended up dropping this suffix for pg_auto_failover. Is there any change we need to make for that

Reply:
We overwrite the value below. No need to fix anything